keps/0001a-meta-kep-implementation.md

```diff
@@ -0,0 +1,206 @@
+
+As [KEP-1] is currently de facto for the project, we must be careful to make changes to it in an iterative and atomic fashion.
+
+When proposing a KEP, we action over some unit of work, usually some area of code.
+
+In this instance, we treat [KEP-1] as the unit of work. That said, this would be considered a meta-KEP of the meta-KEP.
+
+### Non-Goals
+
+- API Review process
+- Feature request triage
+- Developer guide
+
+## Proposal
+
+### Implementation Details / Notes / Constraints
+
+#### Define
+
+- Refine existing KEP documentation
+- Define KEP [DACI]
+
+#### Organize
+
+- Move KEPs from flat-files to a directory structure:
+```
+├── 001-introduction.md (required)
+├── 002-motivation.md (required)
+├── 003-goals.md
+├── 004-non-goals.md
+├── 005-guide-for-developers.md (required)
+├── 006-guide-for-teachers.md (required)
+├── 007-guide-for-operators.md (required)
+├── 008-guide-for-project-maintainers.md
+├── 009-graduation-criteria.md (required)
+├── 010-implementation-history.md
+├── experience_reports (required)
+│   ├── alpha-feedback.md
+│   └── beta-feedback.md
+├── features
+│   ├── feature-01.md
+│   ├── feature-02.md
+│   └── feature-03.md
+├── index.json (required)
+├── metadata.yaml (required)
+└── OWNERS
+```
+
+metadata.yaml contains information that was previously in a KEP's YAML front-matter:
+
+```
+---
+authors: # required
+  - "calebamiles" # just a GitHub handle for now
+  - "jbeda"
+title: "Kubernetes Enhancement Proposal process"
+number: 42 # required
+owning-sig: "sig-pm" # required
+participating-sigs:
+  - "sig-architecture"
+  - "sig-contributor-experience"
+approvers: # required
+  - "bgrant0607" # just a GitHub handle for now
+reviewers:
+  - "justaugustus"  # just a GitHub handle for now
+  - "jdumars"
+editors:
+  - null # generally omit empty/null fields
+status: "active" # required
+github:
+  issues:
+    - null # GitHub url
+  pull_requests:
+    - null # GitHub url
+  projects:
+    - project_id: null
+      card_id: null
+releases: # required
+  - k8s_version: v1.9
+    kep_status: "active"
+    k8s_status: "alpha" # one of alpha|beta|GA
+  - k8s_version: v1.10
+    kep_status: "active"
+    k8s_status: "alpha"
+replaces:
+  - kep_location: null
+superseded-by:
+  - kep_location: null
+created: 2018-01-22 # in YYYY-MM-DD
+updated: 2018-09-04
+```
+
+- Move existing KEPs into [k/features]
+- Create a `kind/kep` label for [k/community] and [k/features]
+  - For `k/community`:
+    - Label incoming KEPs as `kind/kep`
+    - Enable searches of `org:kubernetes label:kind/kep`, so we can identify active PRs to `k/community` and reroute the PR authors to `k/keps` (depending on the state)
+  - For `k/keps` (fka `k/features`):
+    - Label incoming KEPs as `kind/kep`
+    - Classify KEP submissions / tracking issues as `kind/kep`, differentiating them from `kind/feature`
+- Move existing design proposals into [k/features]
+- Move existing architectural documents into [k/features]
+- Deprecate design proposals
```

Comment:
Does this imply from this point forward, everything must be a KEP?

Reply:
Any significant, multiple-cycle work should be a KEP.
Here's a good example of a design proposal, that is essentially a KEP, that should probably just be in the KEPs dir: https://github.com/kubernetes/community/pull/2514